src/caf/distribute/array_utils.py

```diff
@@ -0,0 +1,538 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think any references to array_utils within this package need updating. This probably still passes tests as current released versions of toolkit still contain what's needed.